docs/extensibility/jsonata.md

```diff
@@ -0,0 +1,38 @@
+# Jsonata preset functions for resource-based extensions
+
+**Table of Contents**
+
+- [Overview](#overview)
+- [Preset functions](#preset-functions)
+  - [_matchByLabelSelector_](#matchbylabelselector)
+
+## Overview
+
+This document describes how to use the preset JSONata functions.
+
+## Preset functions
+
+## matchByLabelSelector
```

Comment:
Maybe it would be better to specify the arguments in the header?
```suggestion
## matchByLabelSelector(item, selectorPath)
```

Also please change the second parameter name, as it's not a `label`, but the path to labels.